### PR TITLE
Update Release_notes.md with 7.7.0 version 

### PR DESCRIPTION
**Pull request by SaikrishnaBairamoni**
# PR Details
This PR is to add Release notes for 7.7.0 and updated previous release notes 7.6.0
## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
[V2XHUB Contributing Guide](https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/develop/Contributing.md) 
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by maefromm — COMMENTED** [review]
@SaikrishnaBairamoni I am confused with the versioning. It seems we had 7.6.0 as our next release for August, however I see it's been used already? 
Master Release Notes https://github.com/usdot-fhwa-OPS/V2X-Hub/blob/master/docs/Release_notes.md shows 7.5.1

File: docs/Release_notes.md [hunk 1]
```diff
@@ -1,6 +1,59 @@
 V2X-Hub Release Notes
 ---------------------------------
 
+Version 7.7.0, released Aug 14th, 2023
+--------------------------------------------------------
+
+**Summary:**
+V2X Hub release 7.7.0 introduces key enhancements and fixes to improve data streaming, upgrade MySQL to version 8.0, and support multi-architecture Docker deployments. This release includes updates to initialization scripts and addresses clock reference issues in simulation mode. Additionally, this release removes obsolete Docker tags and makes the SENSOR_JSON_FILE_PATH an optional environment variable to enhance flexibility in simulation setups.
+
+**<ins>V2X Hub CDASim Functionalities </ins>**
```
Comment (maefromm) [inline docs/Release_notes.md:10]: Remove this

File: docs/Release_notes.md [hunk 2]
```diff
@@ -1,6 +1,59 @@
...
+
+Enhancement in this release:
```
Comment (maefromm) [inline docs/Release_notes.md:12]: Match the rest with how we presented to the client.

File: docs/Release_notes.md [hunk 3]
```diff
@@ -1,6 +1,59 @@
...
+
+Other Enhancements: 
+- V2X-Hub PR 612: Updated MySQL to 8.0 to support multi-architecture Docker deployment, removed architecture-based configuration folders, and updated initialization scripts.
+- V2X-Hub PR 615: Updated the initialization script to generate a .env file based on user input.
+  
+Other Fixes:
+- V2X-Hub PR 614: Removed the "version:" tag at the top of the docker-compose files. These are no longer needed and are obsolete in current and future versions of Docker.
+
+
+Version 7.6.0, released April 16th, 2024
```
Comment (maefromm) [inline docs/Release_notes.md:26]: Based on the Platform release notes and on SharePoint, this should be April 10th, not 16th? 

I see that 7.6.0 release notes were missed somehow for Lavida. So good with this as long as it matches what we sent the client for approval.

**Review by maefromm — CHANGES_REQUESTED** [review]
@SaikrishnaBairamoni  Please match the content to what we have sent the client.

File: docs/Release_notes.md (hunk 1 above)
Comment (SaikrishnaBairamoni) [reply docs/Release_notes.md]: Updated it SharePoint document

File: docs/Release_notes.md (hunk 2 above)
Comment (SaikrishnaBairamoni) [reply docs/Release_notes.md:10]: updated

File: docs/Release_notes.md (hunk 3 above)
Comment (SaikrishnaBairamoni) [reply docs/Release_notes.md]: I've added 7.6.0 notes from client approval document in Share Point.

**Review by maefromm — APPROVED** [review]

**Pull request merged**